www/inc/header.php

```diff
@@ -36,7 +36,7 @@
     <script>jQuery.noConflict();</script>
     <script src="//cdnjs.cloudflare.com/ajax/libs/angular.js/1.1.5/angular.min.js"></script>
     <script src="//cdnjs.cloudflare.com/ajax/libs/angular-ui/0.4.0/angular-ui.min.js"></script>
-    <script src="//w3.scripts.mit.edu/rdflib.js/dist/rdflib.js" type="text/javascript"></script>
+    <script src="/dist/rdflib.min.js" type="text/javascript"></script>
```

Comment:
This is the `rdflib.js` link is broken.
It seems to build this JS library from this [link](https://github.com/linkeddata/rdflib.js) by yourself.